aws/cloudformation/drone-stack.yml

```diff
@@ -261,7 +261,8 @@ Resources:
         KeyName: !Sub "winter-dev-${AWS::Region}"
         SecurityGroupIds: 
           - !Ref DroneWorkerEcsSecurityGroup
-        ImageId: ami-0c4b05c0409735671
+        # ECS Optimized AMI - https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html
+        ImageId: ami-04bb74f3ffa3aa3e2
```

Comment:
Drone Server 1.9 issues Docker API calls that require a newer version of Docker, which is available by upgrading to the latest version of ECS-optimized (docker pre-installed) Amazon Linux.